### PR TITLE
Video Library v1 #PR1

### PR DESCRIPTION
**Pull request by suyashpradhan**

### CODE REVIEW

**Review by rajat-mehra05 — APPROVED** [review]

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
```
Comment (Gagan-Deogan) [inline public/index.html:6]: you can add FavIcon

File: public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (Gagan-Deogan) [inline public/manifest.json:1]: you can delete this file as you are not using it

File: src/Components/Header/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
+import React from "react";
```
Comment (Gagan-Deogan) [inline src/Components/Header/Header.jsx:1]: As your using JSX , there is no need to always import whole react libaray

**Review by Gagan-Deogan — COMMENTED** [review]
Code is Very well, here are my suggestion and you forget to add _redirect file which will cause problem in production if user reload the app it will redirect to 404 page

**Comment by suyashpradhan** [comment]
> Code is Very well, here are my suggestion and you forget to add _redirect file which will cause problem in production if user reload the app it will redirect to 404 page

Thank you, will update the changes

**Comment by Sohail737** [comment]
Code is very well structured, looks good

**Pull request closed without merging**